### DISCUSSION TITLE
Use pyodide.http.pyFetch in .py file

### DISCUSSION THREAD

**Discussion by KXTOD** [pyodide/pyodide#3320] [Q&A]
This might be a stupid question, but I can't figure out how to use pyFetch in a .py file. I'm using this line `<py-script src="/get_top_items.py"></py-script>` to import the python file and the code in there uses requests. Turns out requests is not compatible, so my question is: how do I make an https request to a server inside the `get_top_items.py` file.

Would love some help.

(also tried searching if anyone had asked but this but could not find it)

**Comment by loosak** [comment]
You should rtfm [here](https://docs.pyscript.net/latest/howtos/http-requests.html) ....

**Reply by rth** [reply]
Thanks for the information, but this is not very welcoming/OK wording.

In addition to the information on the linked page, there is https://github.com/koenvo/pyodide-http for drop-in replacement if needed.
